src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -581,6 +581,10 @@ def test_fw_gen_use(fortran_writer):
     assert "use my_module, only : dummy1=>orig_name, my_sub" not in result
     assert "use my_module\n" in result
 
+    container_symbol.intrinsic = True
+    result = fortran_writer.gen_use(container_symbol, symbol_table)
+    assert "intrinsic" in result
```

Comment:
Can you compare the test of the whole use statement please?

Reply:
Added